tests/integration/common_workflows/test_sql_asset_workflows.py

```diff
@@ -109,6 +109,55 @@ def _create_test_cases():
     ]
 
 
+def test_get_batch_identifiers_list__simple(
```

Comment:
This test requires the DB to be seeded already; TODO: Fix in another PR.

Reply:
Can we leave a comment about this?